ghost/i18n/locales/tr/portal.json

```diff
@@ -95,8 +95,8 @@
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "Haber bülteni göndermeye çalışırken kalıcı bir hata alınması durumunda, hesapta e-postalar devre dışı bırakılır.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "E-posta istemcinizde {{senderEmail}} adresini kişi listenize ekleyin. Bu, posta sağlayıcınıza bu adresten gönderilen e-postaların güvenilir olması gerektiğini bildirir.",
     "Invalid email address": "Geçersiz e-posta adresi",
-    "Jamie Larson": "Ahmet Yılmaz",
-    "jamie@example.com": "ahmet@example.com",
+    "Jamie Larson": "Ad Soyad",
```

Comment:
Could you please provide some context for this change?

Reply:
I double checked, and the original translation is more inline with our intention (a pretend, generic name) and the latter literally means "First name Surname". Therefore I think this update is incorrect.

Reply:
I'm sure everyone will use this, and no one would want someone else's name and surname to appear on their site. For this reason, when a name and surname are entered, the registered user will see their own name and surname. Entering a name and surname is the correct approach.

Reply:
You've misunderstood what this string is used for.  It's a placeholder.  It goes into the subscribe box until the user types their name.  Once they type their name, that's what appears there.  When a name and surname are entered, the user will indeed see their own name in the subscription box.  But when they first load the page, they should see an 'example' name.  Here's how it looks in English.  Notice the names are greyed out:
![image](https://github.com/user-attachments/assets/ffcd6022-3a64-44d1-9f8b-5348e5b5690f)
Then, after I type in my information, here's how it looks:
![image](https://github.com/user-attachments/assets/d332ea33-3ba9-418b-99eb-5052d6ac31cc)

Reply:
I am fully aware of what it is, but the current translation is incorrect. Therefore, I made the necessary correction.